.golangci-kal.yml

```diff
@@ -149,81 +153,29 @@ linters:
       linters:
```

Comment:
Trying to fix this finding in https://github.com/kubernetes-sigs/cluster-api/pull/12565